21/README.md

```diff
@@ -0,0 +1,60 @@
+```
+shortname: 21/PYTHON-ORM-DRIVER
+name: Orm implementation for official bigchaindb python driver
```

Comment:
Please write the name in title case: ORM Implementation for Official BigchainDB Python Driver